tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -239,16 +239,57 @@ export function filterByKind(apiItems: readonly ApiItem[], kinds: ApiItemKind[])
 }
 
 /**
- * Gets the release tag associated with the provided API item, if one exists.
+ * Gets the release tag associated with the provided API item, if the item's documentation contained one.
  *
- * @param apiItem - The API item whose documentation is being queried.
+ * @returns The associated release tag, if it exists. Will return `None` if no tag is present.
```

Comment:
`@param` docs want to come back, I think?

Reply:
I actually removed it intentionally because it felt redundant with the function summary. But we do include it elsewhere. I'll restore it (slightly modified) for consistency.